app/models/event_invoice.py

```diff
@@ -97,6 +99,7 @@ def __init__(self,
         self.last4 = last4
         self.stripe_token = stripe_token
         self.paypal_token = paypal_token
+        self.deleted_at = deleted_at
```

Comment:
But why are you adding this property when it inherits from SoftDeletionModel?

Reply:
I am not adding it. It is already declared from the base SoftDeletionModel, I am initialising it. Am I wrong in this ? @schedutron @mayank8318